### ISSUE TITLE
Show txoutproof of transaction [for Liquid Peg-In]

### ISSUE THREAD

**Issue by FrancisPouliot** [Blockstream/esplora#169]
When doing a peg-in via Liquid, it is necessary to have the txoutproof in order to claim the pegin. It would be really helpful to have that available in the explorer, because it requires a synced fullnode and not always available. 

Edit: might as well show the transaction hex as well

Thanks!

**Comment by tiero** [comment]
I also need this 🙏

**Comment by shesek** [comment]
The esplora API does [provide merkle proofs](https://github.com/Blockstream/esplora/blob/master/API.md#get-txtxidmerkle-proof), but using the Electrum merkle proofs format and not bitcoind's.

Back when this was implemented in esplora, there wasn't a Rust implemention for bitcoind-style `merkleblock` proofs -- but I just looked again now, and it seems like [it was since added](https://github.com/rust-bitcoin/rust-bitcoin/blob/master/src/util/merkleblock.rs) to rust-bitcoin! Thanks for nudging me to look again :)

Adding this to esplora should be pretty straightforward, I will look into doing this.

Duplicate of #24, will continue tracking this there.
